src/app/charts/chart-base-bar/chart-base-bar.component.ts

```diff
@@ -72,7 +72,10 @@ export class ChartBaseBarComponent extends ChartBaseComponent {
       .attr('class', () => (this.categorical ? 'bar' : 'bar-time'))
       .attr(
         'x',
-        d => (this.categorical ? this.xScale(d.name) : this.xScale(d.date))
+        d =>
+          this.categorical
+            ? this.xScale(d.name)
+            : this.xScale(d.date) - this.width / this.data.length / 2
```

Comment:
I'm would like to understand why `this.xScale(d.date) - this.width / this.data.length / 2` computation is made for XScale. Maybe some documentation in the code would help

Reply:
This was to center the bar of the bar charts to the specific date in the x axis. But I've removed this because it's more intuitive if the bar starts at the date since it's an interval.